setup.py

```diff
@@ -13,20 +13,26 @@
 requires = [r.strip() for r in req_data.split() if r.strip() != '']
 requires = list(reversed(requires))
 
+# testing dependencies
+req_data = open(os.path.join(__here__, 'test_requirements.txt')).read()
+tests_require = [r.strip() for r in req_data.split() if r.strip() != '']
+tests_require = list(reversed(tests_require))
+
 # temporary workaround for a Python 2 CherryPy bug, for which we opened a pull request:
 # https://bitbucket.org/cherrypy/cherrypy/pull-request/85/1285-python-2-now-accepts-both-bytestrings/
 if sys.version_info[0] == 2:
     requires = ['CherryPy==3.2.2' if 'cherrypy' in r.lower() else r for r in requires]
 
 if __name__ == '__main__':
+    setup_requires = {'setup_requires': ['distribute']} if sys.version_info[0] == 2 else {}
     setup(
         name=pkg_name,
         version=__version__,
         description='Sideboard plugin container.',
         license='BSD',
         scripts=[],
-        setup_requires=['distribute'],
         install_requires=requires,
+        tests_require=tests_require,
```

Comment:
I still like the idea we discussed last time this came up, which is:
- ``sideboard.tests`` remains a module, but it *only* contains test utilities which are deliberately exposed to plugins as part of its API
- all regular unit tests for Sideboard get moved into a ``tests/`` directory

I'll write this up into an actual ticket this week and see if I can take a stab at actually doing it, since I'd prefer to have that done sooner rather than later, since otherwise it'll create a bunch of merge conflicts if one branch has that change and another doesn't.